doc/guides/platformio/index.rst

```diff
@@ -0,0 +1,61 @@
+.. _platformio:
+
+Using with PlatformIO
+#####################
+
+.. image:: img/platformio-logo.png
+  :target: https://platformio.org/?utm_source=docs.zephyrproject.org
+
+.. contents::
+    :local:
+
+What is PlatformIO?
+*******************
+
+`PlatformIO <https://platformio.org/?utm_source=docs.zephyrproject.org>`__
+is a cross-platform, cross-architecture, multiple framework tool for embedded
+systems engineers and software developers who write applications for embedded
+products. It is fully compatible with the Zephyr project.
```

Comment:
```suggestion
is a cross-platform embedded development environment with
Zephyr support maintained by its developers.

Since Zephyr support within PlatformIO is not maintained by the
Zephyr Project, please report any issues with PlatformIO directly to
its developers.
```

Reply:
If you can add a link to how you want to get bug reports etc. to the text here, IMO that would be even better.